src/TodoApp.ts

```diff
@@ -134,11 +163,32 @@ export class TodoApp extends LitElement {
 
   addTodo() {
     if (this.newTodo.trim() !== '') {
-      this.todos = [...this.todos, this.newTodo.trim()];
-      this.newTodo = '';
-    }
+        this.todos = [...this.todos, { text: this.newTodo.trim(), completed: false }];
+        this.pendingTasksCount++;
+        this.updatePendingTasksCount();
```

Comment:
line 167 and 168 are doing the same thing ?

Reply:
**Yeah Did the necessary changes**